.travis.yml

```diff
@@ -1,18 +1,18 @@
 language: java
+dist: trusty
```

Comment:
Is there a specific reason for using `trusty`?

Reply:
The build was originally failing because the [default build environment changed from `trusty` to `xenial`](https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment) so now we have to specify it.